qiskit_nature/algorithms/ground_state_solvers/minimum_eigensolver_factories/numpy_minimum_eigensolver_factory.py

```diff
@@ -45,28 +44,49 @@ def __init__(
         """
         self._filter_criterion = filter_criterion
```

Comment:
Do we even still need this private attribute or could this be replaced by `self._minimum_eigensolver.filter_criterion` everywhere?

Reply:
Removed